plugins/scaffolder-backend/sample-templates/create-react-app/template.yaml

```diff
@@ -20,6 +20,7 @@ spec:
           title: Name
           type: string
           description: Unique name of the component
+          ui:field: EntityNamePicker
```

Comment:
Can't update the templates before the UI component has been released.

Reply:
Makes sense! Will be nice to get rid of these templates soon. Will split this off :)

Reply:
#6726.